.NET/Microsoft.Recognizers.Text.DateTime/Utilities/RangeTimexComponents.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.Text;
+using DateObject = System.DateTime;
+
+namespace Microsoft.Recognizers.Text.DateTime
+{
+    public class RangeTimexComponents
```

Comment:
On the fields of this class, VS is reporting warning [SA1401](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1401.md) "Field should be private". I think they missing `{ get; set; }`.

Reply:
Feedback applied.